app/components/failed_network_action/cloud.js

```diff
@@ -16,7 +16,10 @@ export default class CloudSvg extends PureComponent {
     render() {
         const {color, height, width} = this.props;
         return (
-            <View style={{height, width, alignItems: 'flex-start'}}>
+            <View
+                style={{height, width, alignItems: 'flex-start'}}
+                testID='cloud_icon'
+            >
                 <Svg
```

Comment:
@matthewbirtch should we replace this with  a composeIcon ?

Reply:
We don't really have a suitable replacement right now, so let's leave it for now